### DISCUSSION TITLE
How to configuring Yagi to adjust window size when moving boundaries.

### DISCUSSION THREAD

**Discussion by 10bn** [asmvik/yabai#1562] [Q&A] [answered]
Is it possible to configure Yagi to automatically adjust the size of windows that are already touching each other when moving window boundaries, similar to the native split-screen feature on macOS and switch all other features of?

**Comment by asmvik** [comment, marked as answer]
This is a built-in feature that is always enabled -- It only works/makes sense when windows are managed in a bsp-layout:



![mouse_adjust](https://user-images.githubusercontent.com/6175959/208091705-73991bff-7a2f-413b-ab51-7e3fd1474331.gif)

**Reply by 10bn** [reply]
Hello, thanks for the example, but I only want it to behave like this if the windows are already snapped together. Currently it auto aranges all open windows and I'm unable to turn that off. 

Is this also possible, to have the mutal resizing only for windows which are already border to border?

**Reply by asmvik** [reply]
This is only possible because windows are auto-arranged.
